### PR TITLE
Fix a link

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -1427,7 +1427,7 @@ console) are redirected to GitHub to log in.
 <4> Controls how mappings are established between this provider's identities and user objects,
 xref:mapping-identities-to-users[as described above].
 <5> The client ID of a
-link:https://github.com/settings/applications/new[registered GitHub OAuth
+link:https://docs.gitlab.com/ce/api/oauth2.html[registered GitHub OAuth
```
Comment (kalexand-rh) [inline install_config/configuring_authentication.adoc]: I think the original link makes more sense.

Comment (vikram-redhat) [reply install_config/configuring_authentication.adoc]: @kalexand-rh I am gonna merge this. If you still feel that the original one makes more sense, feel free to change back.

Comment (kalexand-rh) [reply install_config/configuring_authentication.adoc]: @enj, which link is right?

Comment (enj) [reply install_config/configuring_authentication.adoc]: This change is incorrect.  Please restore the original link.

Comment (kalexand-rh) [reply install_config/configuring_authentication.adoc]: Fixing on https://github.com/openshift/openshift-docs/pull/9117

**Pull request merged**